src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -2100,6 +2162,19 @@ public ListBoxModel doFillNotificationLevelItems(@QueryParameter("serverName") f
         }
 
         /**
+         * Fill the dropdown for silent levels.
+         * @return the values.
+         */
+        public ListBoxModel doFillSilentLevelItems() {
```

Comment:
I don't think this is needed if the databinding is performed correctly, use `f:enum`.